index.js

```diff
@@ -0,0 +1,70 @@
+// Create Dom element
+
+const addBooks = localStorage.getItem('adbooks');
+let books = [];
+
+if (addBooks) {
+  books = JSON.parse(addBooks);
+} else {
+  const bookData = [
+    {
+      id: 1,
+      name: 'Lorem ipsum',
+      author: 'Testeroo Testyy',
+    },
+  ];
+
+  localStorage.setItem('adbooks', JSON.stringify(bookData));
+  books = JSON.parse(addBooks);
+  window.location.reload();
```

Comment:
Please note that at the moment your books are not been saved in the local storage. When I refresh the browser all the books are removed from the local storage. Kindly fix that according to the project requirement.

Reply:
Done